### PR TITLE
Display single reminder

### LINKED ISSUE #3: See an individual reminder
The list of reminders should only display Titles.

When a user clicks on one of the existing titles, they are taken to an individual reminders route (ie: `'/reminders/1'`).

**Hints**:

* Ember convention uses `:model_id` as the dynamic path, ie: '/reminders/:reminder_id'.
* You can find help [here](https://guides.emberjs.com/v2.12.0/routing/specifying-a-routes-model/#toc_dynamic-models)
* Run `ember g route reminders/reminder`
* The user should still see all of the reminders from your previous feature. The specific reminder they are looking at should now be rendered in the `outlet` of the `reminders.hbs` template.
* There should be some CSS for the `.active` class.
**Hot Tip**: There is an existing acceptance test for this feature.

### PR DESCRIPTION
**Pull request by becs919**
@martensonbj @Tman22 
List of reminders displays only titles
When clicked on existing title, taken to new route based on id
All reminders still render
Clicked reminder renders below
Add some CSS
Acceptance test passing for clicked reminder
closes #3 

## Purpose
Allow the user to click a reminder title and see the body and date rendered below the title list. 

## Approach
We first removed everything but the title from rendering on the root page. We then added a path that corresponded to the reminder's id that was clicked. Added a link to the titles so they could be clicked. We struggled with the test but Taylor helped us with that one, we had to play around with class names. 

### Learning
A lot of docs research, specifically looking at the tutorial and https://guides.emberjs.com/v2.12.0/routing/specifying-a-routes-model/#toc_dynamic-models
Also thanks to help from both Taylor and Brenna 👍 

### Test coverage 
Passed acceptance test that was already provided 

### Follow-up tasks
issue #4 

### Screenshots
after

### CODE REVIEW

File: app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+body {
+  font-family: 'Playfair Display', serif;
+}
+
+.reminders-div {
+  border-right: 2px solid black;
+  color: slategray;
+  padding: 20px;
+  width: 25%
+
```
Comment (martensonbj) [inline app/styles/app.css]: Remove extra blank line please

File: tests/unit/routes/reminders/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
```
Comment (martensonbj) [inline tests/unit/routes/reminders/reminder-test.js]: Remove unnecessary test file

**Review by martensonbj — CHANGES_REQUESTED** [review]
Couple small clean up things - nice work guys

**Comment by becs919** [comment]
@martensonbj @Tman22 
Fixed the spacing in the css file and deleted the unneeded test.

**Comment by martensonbj** [comment]
❤️

**Pull request merged**